comment_iowriter.go

```diff
@@ -56,3 +59,86 @@ func (mc *UnifiedCommentWriter) Post(_ context.Context, c *Comment) error {
 	_, err := fmt.Fprintln(mc.w, s)
 	return err
 }
+
+var _ CommentService = &RDJSONLCommentWriter{}
+
+// RDJSONLCommentWriter
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>comment on exported type RDJSONLCommentWriter should be of the form "RDJSONLCommentWriter ..." (with optional leading article)
<!-- __reviewdog__:ChA2YTEwNDc5NWE2ODMzYWU4Eg9nb2xpbnQtY2lyY2xlY2k= -->